client/health.go

```diff
@@ -18,30 +18,22 @@ import "net/url"
 
 // HealthOptions holds query options to pass to a Health call.
 type HealthOptions struct {
-	// Level is the check level to filter for. A check is included in the
-	// healthiness determination if this field is not set, or if it is
-	// equal to the check's level.
+	// Level may be set to CheckAlive, to query whether alive checks are up, and
+	// CheckReady, to query whether both alive and ready checks are up.
+	// Defaults to CheckReady if unset.
 	Level CheckLevel
 
-	// Names is the list of check names to filter for. A check is included in
-	// the healthiness determination if this field is nil or empty slice, or
-	// if one of the values in the slice is equal to the check's name.
+	// Names defines which checks should be considered for the query. Defaults to all.
 	Names []string
 }
 
-// HealthInfo holds the result of a Health call.
-type HealthInfo struct {
-	// Healthy is the status of health. A set of checks are deemed "healthy"
-	// if all of them of are up, and "unhealthy" otherwise. When queried
-	// using level, if the queried level equals "alive", only the alive
-	// checks are selected. However, "ready" implies alive. Thus, if the
-	// queried level is "ready", both the alive and ready checks are
-	// considered.
+// healthInfo holds the result of a Health call.
```

Comment:
This comment is no longer quite correct, as `Health` now returns a bool. Could probably just drop the comment now.

Reply:
Nice catch! Thanks, dropped.